src/checks/stylelint.ts

```diff
@@ -0,0 +1,40 @@
+import { execaCommand } from 'execa';
+import { LintResult } from 'stylelint';
+
+export const supportedExtensions = ['css', 'scss', 'less', 'js', 'tsx', 'jsx'];
+
+export const title = 'Stylelint';
+
+export default async function stylelintCheck() {
+  try {
+    await execaCommand(
+      `yarn stylelint **/*.{${supportedExtensions.toString()}} --max-warnings 0 --formatter json`,
```

Comment:
let's be explicit about the joining with `.join()`

Reply:
https://github.com/upleveled/preflight/pull/377/commits/1de52dfb3e47beab9affdf4243db5a1c45fed0b0

Reply:
with an argument

Reply:
01b4935

Reply:
01b4935